### ISSUE TITLE
Tour of how to use PublicLab.org

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#441]
StackOverflow has a great tour of how their community works:

http://stackoverflow.com/tour

It's text and images, not too long, and is pretty cool. We could do this as a wiki page. @ebarry - maybe related to the list you keep of "new community features" or something, and to the request @steviepubliclab made for a wizard in Spectral Workbench.

![screenshot 2016-03-23 at 2 42 14 pm](https://cloud.githubusercontent.com/assets/24359/13996788/9909eb16-f105-11e5-9be4-478c90462947.png)

**Comment by ebarry** [comment]
Yes awesome. We've talked about doing this for a while. I would prefer this be a wizard (like on WhereWeBreathe) and not a wiki page.

**Comment by jywarren** [comment]
It's easier to develop it and edit it as a wiki page, then to reformat it
into a wizard. That way we can develop and iterate the content without
needing to code yet. Is that ok?

**Comment by jywarren** [comment]
https://publiclab.org/wiki/website-basics and wizard could be done as followup
